src/main/java/hudson/plugins/git/GitPublisher.java

```diff
@@ -573,15 +573,15 @@ public String getDisplayName() {
         private String noteNamespace;
         private boolean noteReplace;
 
-        public String getnoteMsg() {
+        public String getNoteMsg() {
```

Comment:
Breaks backward compatibility for a class that is public API. Did you confirm none of the ~60 plugins with dependency on the Git Plugin use these methods?